### ISSUE TITLE
💅 Migrate drawer to use Tailwind UI Sidebar

### ISSUE THREAD

**Issue by asim-shrestha** [reworkd/AgentGPT#672]
https://tailwindui.com/components/application-ui/application-shells/sidebar

We should still keep the same buttons / layout / etc, but leverage this more reusable sidebar

**Comment by Alimjan2013** [comment]
It is free to use? I see a price in tailwind UI website

**Comment by jasangill1** [comment]
@Alimjan2013 yes I believe you must get the paid version

**Comment by asim-shrestha** [comment]
Yeah, we use this internally so mostly for our tracking. Soryy @Alimjan2013 !
